docker/Dockerfile

```diff
@@ -1,13 +1,15 @@
-FROM golang:1.14
-ENV GOBIN /go/bin
-RUN go get github.com/stellar/stellar-etl
+FROM golang:1.16
+
+COPY . /etl
```

Comment:
we weren't installing from local before. I think that's why it's giving build errors of trying fetch remotely. Not sure if that was intentional, but I think these changes are what we want anyways though. It follows the pattern of other Go Dockerfiles we have